packages/components/src/Checkbox/Checkbox.tsx

```diff
@@ -175,6 +175,13 @@ export const Label = ({
 };
 
 /**
+ * ```ts
+ * import {Checkbox} from "@chanzuckerberg/eds-components";
+ * ```
+ * ```ts
+ * import Checkbox, {CheckboxInput, Label} from '@chanzuckerberg/eds-components/lib/Checkbox';
+ * ```
```

Comment:
Needs to be separated like this or weird styling issues occur:
![single code block](https://user-images.githubusercontent.com/86632227/154617431-06976ead-e7e4-40ca-9dde-0d58125a3bd4.png)